packages/runner/src/lib/dom.js

```diff
@@ -112,6 +112,11 @@ function addElementBoxModelLayers ($el, $body) {
       obj.left -= dimensions.marginLeft
     }
 
+    if (attr === 'Padding') {
```

Comment:
why is this necessary? what test would have failed without this?

Reply:
both tests would have failed. the Content box (blue) would have been outside the Padding box (green)